vmm/src/device_manager/mmio.rs

```diff
@@ -22,7 +21,7 @@ pub enum Error {
     /// Failed to perform an operation on the bus.
     BusError(devices::BusError),
     /// Could not create the mmio device to wrap a VirtioDevice.
-    CreateMmioDevice(sys_util::Error),
+    CreateMmioDevice(io::Error),
```

Comment:
You also have to change the implementation of `Display` and use to_string instead of debug in formatting.

Reply:
done